### DISCUSSION TITLE
Problema con el archivo de word

### DISCUSSION THREAD

**Discussion by DaviiidSG** [Launch-X-Latam/MisionFrontEnd#133] [Q&A]
Para editar el archivo Requerimientos.doc (en la nube porque no tengo la app de word) me obliga a convertirlo en un archivo más reciente (.docx) pero al realizar la conversión se cambian varias cosas en el archivo y no puedo trabajar bien con él, alguna recomendación para editarlo sin problemas?
![image](https://user-images.githubusercontent.com/110501225/193638039-4c0fdcf4-03c6-4c2f-8170-199f5702958c.png)

**Comment by FaustoPaFl** [comment]
Una posibilidad, no se si funcione es que uses google docs, descarga el archivo y súbelo a la nube de tu google docs, intenta esa opción.

Por otra parte existe un software de edición de documentos estilo MSO, te los enlistan en esta página, espero haber ayudado:

https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwj_ldSt6cT6AhX7JUQIHXjADjQQFnoECAkQAw&url=https%3A%2F%2Fwww.xataka.com%2Fbasics%2Falternativas-gratis-a-word-para-editar-y-crear-documentos&usg=AOvVaw1VdzjmzWLLeFU6wVgctvXj

**Reply by FaustoPaFl** [reply]
jajaja disculpa mi redacción iba ideando el texto conforme pensaba jajaja saludos y suerte

**Comment by ElPoder1** [comment]
Yo lo estoy modificando igual en la nube de microsoft 365 y me aparece igual, para editar se muestra asi como lo tienes, pero cuando cambias a "Vista de lectura" ya aparece normal
